jsonschema-core.md

```diff
@@ -400,14 +400,16 @@ tokens:
 
 - individual Unicode characters, as defined by the [JSON
   specification](#rfc8259);
-- simple character classes ([abc]), range character classes ([a-z]);
-- complemented character classes ([^abc], [^a-z]);
-- simple quantifiers: "+" (one or more), "*" (zero or more), "?" (zero or one),
+- simple character classes ("[abc]"), range character classes ("[a-z]");
+- complemented simple character classes ("[^abc]"),
+  complemented range character classes ("[^a-z]");
+- simple quantifiers: "." (any character except line terminator),
+  "+" (one or more), "*" (zero or more), "?" (zero or one),
   and their lazy versions ("+?", "*?", "??");
 - range quantifiers: "{x}" (exactly x occurrences), "{x,y}" (at least x, at most
   y, occurrences), {x,} (x occurrences or more), and their lazy versions;
 - the beginning-of-input ("^") and end-of-input ("$") anchors;
-- simple grouping ("(...)") and alternation ("|").
+- simple grouping ("(" and ")") and alternation ("|").
```

Comment:
I'm not sure about this.  It seems to indicate the two parentheses can be use independently.

Reply:
How about qualifying with "using"? I.e.,

`- simple grouping (using "(" and ")") and alternation ("|").`

(I made the original change because the grouping doesn't contain the literal "...")

Reply:
I did this in commit 1e16aa4

Reply:
> I made the original change because the grouping doesn't contain the literal "..."

I get it, but this also isn't ABNF.  There's some interpretation here.

I think the "using" is better.